test/unit/logger-test.js

```diff
@@ -0,0 +1,12 @@
+const logger = require("../../server/logger");
+
+describe("Logger", function () {
+  it("It should log error while sending logger.error", function (done) {
+    logger.error("Error");
```

Comment:
This does not do anything. Just test the public functions. They should return a winston.logger